### PR TITLE
Check minimum Keyman Desktop version before bundling

### PR DESCRIPTION
**Pull request by mcdurdin**

### CODE REVIEW

File: resources/util.sh [hunk 1]
```diff
@@ -110,4 +110,23 @@ function parse_args {
...
+
+function verlt {
+  [ "$1" = "$2" ] && return 1 || verlte $1 $2
+}
+
+#verlte 2.5.7 2.5.6 && echo "yes" || echo "no" # no
+#verlt 2.4.10 2.4.9 && echo "yes" || echo "no" # no
+#verlt 2.4.8 2.4.10 && echo "yes" || echo "no" # yes
+#verlte 2.5.6 2.5.6 && echo "yes" || echo "no" # yes
+#verlt 2.5.6 2.5.6 && echo "yes" || echo "no" # no
```
Comment (darcywong00) [inline resources/util.sh]: Is `verlte()` comparing that *a* is "less than, equal to" *b*? 

From these test cases, it seems the function is more like `vergte`

Did you also check this?
```
#verlte 2.5.7 2.5.7 && echo "yes" || echo "no"
```

Comment (mcdurdin) [reply resources/util.sh]: Ugh, not sure how I managed to break that before. Try again now.

File: resources/util.sh [hunk 2]
```diff
@@ -110,4 +110,29 @@ function parse_args {
...
+function verlt {
+  [ "$1" = "$2" ] && return 1 || verlte $1 $2
+}
+
+# verlte 2.5.7 2.5.6 && echo "yes" || echo "no" # no
+# verlt 2.4.10 2.4.9 && echo "yes" || echo "no" # no
+# verlt 2.4.8 2.4.10 && echo "yes" || echo "no" # yes
+# verlte 2.5.6 2.5.6 && echo "yes" || echo "no" # yes
+# verlt 2.5.6 2.5.6 && echo "yes" || echo "no" # no 
+
```
Comment (darcywong00) [inline resources/util.sh]: looks like you can remove lines 128-132 now.
And include one for
```
# if $(verlte 2.5.7 2.5.7) ; then echo "yes" ; else echo "no" ; fi # ...
```

File: ci.sh [hunk 3]
```diff
@@ -222,8 +231,12 @@ function upload_keyboard {
   
   if [[ ${package_filename##*.} == kmp ]]; then
     # We only upload a combined installer for .kmp files
-    create_package_installer "$buildpath" "$buildpath/$installer_filename" "$buildpath/$package_filename" "$package_name" "$package_version"
-    prepare_for_upload "$buildpath/$installer_filename" "$installer_upload_path"
+    if $(verlte "$KEYMANDESKTOP_VERSION" "$min_required_desktop_version"); then
```
Comment (darcywong00) [inline ci.sh]: Now that `verlte` compares a <= b, doesn't this become
```
if $(verlte "min_required_desktop_version" "$KEYMANDESKTOP_VERSION"); then
```

File: resources/util.sh (hunk 2 above)
Comment (mcdurdin) [reply resources/util.sh]: `verlte 2.5.7 2.5.7` is the same as `verlte 2.5.6 2.5.6` is it not? The test cases are commented out, so they are just for reference now; wanted to leave both patterns.

File: ci.sh (hunk 3 above)
Comment (mcdurdin) [reply ci.sh]: Argh. Not my day for comparisons.

File: resources/util.sh (hunk 2 above)
Comment (darcywong00) [reply resources/util.sh]: ic. I just wanted to make sure there were 3 test cases each for `verlt` and `verlte`

**Pull request merged**